ClosedXML.Report/Options/TagsList.cs

```diff
@@ -7,13 +7,16 @@ namespace ClosedXML.Report.Options
 {
     public class TagsList : SortedSet<OptionTag>
     {
-        public TagsList() : base(new OptionTagComparer())
+        private readonly TemplateErrors _errors;
+
+        public TagsList(TemplateErrors errors) : base(new OptionTagComparer())
         {
+            _errors = errors;
         }
 
         public TagsList CopyTo(IXLRange toRange)
         {
-            var clone = new TagsList();
+            var clone = new TagsList(_errors);
```

Comment:
With this, `clone` will receive the reference to the original collection. In this case validation of `clone` will cause the original list's collection of errors to be modified as well. Not sure if it is OK.

Reply:
Logically, it should be so. The error list should be one per template.